docs/MANPAGE_SYNTAX.md

```diff
@@ -29,45 +29,51 @@ podman\-command - short description
+This is an example for 3 (three) arguments behind an OPTION. If the number of arguments is higher than 3 (three), the arguments are **not** listed after the equal sign. The arguments have to be explained in a table like in **--test**=**_test_** regardless of the number of arguments. The default is **good**.
 
-#### **--test**=**_test_**
+#### **--test**=**test**
 
-OPTIONS that are followed by an equal sign include an argument after the equal sign in *italic*. If there is a default argument, that is used if the OPTION is not specified in the **command**, the argument after the eqaul sign is displayed in **bold**. All arguments must be listed and explained in the text below the OPTION.
+OPTIONS that are followed by an equal sign include an argument after the equal sign in **bold**. If there is a default argument, that is used if the OPTION is not specified in the command, the argument after the eqaul sign is displayed in **bold**. All arguments must be listed and explained in the text below the OPTION. The default is **example one**.
 
 | Argument           | Description                                                                 |
 | -                  | -                                                                           |
-| **_example one_**  | This argument is the standard argument if the OPTION is not specified.      |
+| **example one**    | This argument is the standard argument if the OPTION is not specified.      |
 | *example two*      | If one refers to a command, one should use **bold** marks.                  |
-| *example three*    | Example: In combination with **podman command** highly effective.            |
+| *example three*    | Example: In combination with **podman command** highly effective.           |
 | *example four*     | Example: Can be combined with **--exit**.                                   |
 | *example five*     | The fifth description                                                       |
 
 The table shows an example for a listing of arguments. The contents in the table should be aligned left. If the content in the table conflicts with this, it can be aligned in a way that supports the understanding of the content. If there is a standard argument, it **must** listed as the first entry in the table.
 
 If the number of arguments is smaller than 4 (four) they have to be listed behind the OPTION as seen in the OPTION **--status**.
 
+#### **--problem**=*container*
+
+OPTIONS that are followed by an equal sign that is followed by an unspecified argument, have no standard argument. If this OPTION is written with an equal sign and the argument is left empty, there will be no error, but the OPTION will be ignored. The meaning of the argument is described preferably in `one` word after the equal sign in *italic* format.
```

Comment:
I'm a bit lost on this one.  I'm seeing `*container*` here, does this description pertain to `--problem`?
```suggestion
OPTIONS that are followed by an equal sign that is then followed by an unspecified argument, have no standard argument. If this OPTION is written with an equal sign and the argument is left empty, there will be no error, but the OPTION will be ignored. The meaning of the argument is described preferably in `one` word after the equal sign in *italic* format.
```